flipperdevices/flipperzero-firmware
# NFC: Fix MFC key invalidation

# What's new

- Check key before invalidation, fixes #2880 and https://t.me/flipperzero/103919/182015

- Missing furi_hal_nfc_sleep after checking A key

- Read B key for each sector, might give more keys

# Verification 

- Try reading some Chinese clones (can't reproduce, but the person who ran into it said it's fixed)

- Try reading card with ACL 7F0788, should find B key

# Checklist (For Reviewer)
